phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataProtocol.java

```diff
@@ -381,10 +371,10 @@ public static MetaDataMutationResult constructFromProto(MetaDataResponse proto)
               result.functions.add(PFunction.createFromProto(function));
           }
           if (proto.getTablesToDeleteCount() > 0) {
-            result.tableNamesToDelete =
+            result.mutatedTableNames =
                 Lists.newArrayListWithExpectedSize(proto.getTablesToDeleteCount());
```

Comment:
I think it should be possible to rename this field in the MetaDataResponse proto as well without affecting backward compatiblity (see https://developers.google.com/protocol-buffers/docs/proto3) WDYT?

Reply:
Yep,  it will not affect backward compatibility.